src/main/scala/cromwell/engine/EngineFunctions.scala

```diff
@@ -0,0 +1,28 @@
+package cromwell.engine
+
+import cromwell.binding.IoInterface
+import cromwell.binding.expression.WdlStandardLibraryFunctions
+import cromwell.binding.values._
+
+import scala.language.postfixOps
+import scala.util.{Success, Try}
+
+/**
+ * Default implementation of Wdl Standard Library functions executed at the workflow level.
+ * This class is abstract to enforce that Backends extend it and customize the behavior if needed.
+ */
+abstract class WorkflowEngineFunctions(override val interface: IoInterface, val context: WorkflowContext) extends WdlStandardLibraryFunctions {
+
+  override def tempFilePath: String = context.root
+  override def globPath(glob: String): String = context.root
```

Comment:
Yeah I'm not a fan of this either. The idea was to 'factorize' the "glob" function like the others in `WdlStandardLibraryFunctions`, because it's almost always the same code. In short that's a way to tell the `glob` function in `WdlStandardLibraryFunctions` where to glob from, which differs from backend to backend. But it's kind of hacky, I'll see if I can find another way or just have each backend implementation override this with its own implementation.